### PR TITLE
implemented Ratings module

### PR DESCRIPTION
**Pull request by hyukkim0304**
@allisonc07 ready for review!
basic structure of Ratings component

### CODE REVIEW

File: client/src/components/ratings/ratings.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const React = require('react');
+
+const Ratings = (props) => {
+  return (
+    <div>
+    Ratings
+      <div>
+      Accuracy
+        {props.data.length !== 0
+          && Math.round((props.data.map(data => data.accuracy)
```
Comment (allisonc07) [inline client/src/components/ratings/ratings.jsx]: is there a way to pull this out as a function that you have above the return statement? so something like

`const getAverage = (reviewField) => 
  Math.round((props.data.map(data => data.reviewField)
    .reduce((acc, cur) => acc + cur)) / props.data.length);
`

and then call it in each field: 
`{props.data.length !== 0
  && getAverage('accuracy')}`

**Review by allisonc07 — COMMENTED** [review]
Left one suggestion that would help reduce repetitiveness in your code! Otherwise, 👍

File: client/src/components/ratings/ratings.jsx (hunk 1 above)
Comment (hyukkim0304) [reply client/src/components/ratings/ratings.jsx]: good idea! ill refactor1

**Review by allisonc07 — APPROVED** [review]
Great!!

**Pull request merged**